oomph-filter-widgets.js

```diff
@@ -0,0 +1,34 @@
+(function($) {
+	if(!window.Oomph) window.Oomph = {};
+
+	Oomph.FilterWidgets = {
+		init: function() {
+			$('#available-widgets .widget-holder p:first-child')
+				.after(
+					$('<p class="oomph-filter-widgets description" title="Search for widgets (Ctrl-Shift-F)"></p>').append(
+						$('<label>').text(Oomph.text.filterWidgets)
+						.append($('<input id="widget-search" type="text">').attr('placeholder', Oomph.text.searchForWidgets))));
+
+			var $search = $('#widget-search');
+			var $widget_list = $('#widget-list');
+
+			$('body').on('keyup', function(ev) {
+				if(ev.keyCode == 70 && ev.ctrlKey)
```

Comment:
Meh, not as bent on that in JS as I am in PHP. it's mostly for debugging in PHP.